backend/hitas/views/apartment.py

```diff
@@ -615,3 +620,34 @@ def get_list_queryset(self):
             )
             .order_by("id")
         )
+
+    @action(detail=True, methods=["GET"])
+    def unconfirmed_prices_pdf(self, request, **kwargs) -> Union[HttpResponse, Response]:
+        apartment = self.get_object()
+        apartment_data = ApartmentDetailSerializer(apartment).data
```

Comment:
first instinct says this fetches a quite a bit of extra information that's not really needed here and gets discarded

Reply:
You're correct, but the other way around this would be to write a serializer used only for fetching the necessary data for this, which I see as a bit too preemptive optimisation, so I went with using our already preexisting code to keep things simpler :)